aztec-up/bin/aztec

```diff
@@ -1,11 +1,30 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+# Create an args array to iterate over
+cmd_args=("$@")
+
+# Check if we're running against an Aztec network
+NETWORK=${NETWORK:-}
+# Also check CLI args that take precedence over network vars
+while [[ "${#cmd_args[@]}" -gt 0 ]]; do
+  case "${cmd_args[0]}" in
+  --pxe.network)
```

Comment:
It still feels weird that we have a "global" setting like a network, but it gets namespaced by `pxe`. Anyway, something to think about for later!